shared/chat/conversation/messages/text/container.tsx

```diff
@@ -22,14 +22,23 @@ const getReplyProps = (replyTo, onReplyClick) => {
     username: '',
   }
   switch (replyTo.type) {
+    case 'attachment':
     case 'text':
+      const attachment: Types.MessageAttachment =
+        replyTo.type === 'attachment' && replyTo.attachmentType === 'image' ? replyTo : null
       return replyTo.exploded
         ? deletedProps
         : {
             deleted: false,
             edited: replyTo.hasBeenEdited,
+            imageHeight: attachment ? attachment.previewHeight : undefined,
```

Comment:
:sheep: feels a little redundant to have the switch and also do this. would splitting the cases duplicate too much?